Vagrantfile

```diff
@@ -97,6 +101,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       when "NO_PROXY"; no_proxy = value
       when "HOST_PORT"; host_port = value.to_i
       when "HOST_IP_ADDR"; host_ip_addr = value
+      when "VM_NUM_CPUS"; vm_num_cpus = value
+      when "VM_MEMORY"; vm_memory = value
```

Comment:
Let's do clearer user-facing names for these, e.g. `GUEST_CPUS` and `GUEST_MEMORY_MB`